lib/solver.cpp

```diff
@@ -239,6 +240,117 @@ namespace quda {
     }
   }
 
+  /*
```

Comment:
what's the point of this (since it's commented out)?

Reply:
This was an attempt to move the vector IO to the solver class. I moved them to the Eigensolver class instead.